src/main/java/org/opensrp/web/rest/ClientFormResource.java

```diff
@@ -218,8 +218,8 @@ private ClientFormMetadata getMostRecentVersion(@NonNull String formIdentifier,
         }
 
         String fileContentType = jsonFile.getContentType();
-        if (!(isClientFormContentTypeValid(fileContentType) || isPropertiesFile(fileContentType, jsonFile.getOriginalFilename()))) {
-            return new ResponseEntity<>("The form is not a JSON/Properties/Yaml file", HttpStatus.BAD_REQUEST);
+        if (!(isClientFormContentTypeValid(fileContentType, jsonFile.getOriginalFilename()) || isPropertiesFile(fileContentType, jsonFile.getOriginalFilename()))) {
+            return new ResponseEntity<>("The form is not a JSON/Properties/Yaml file. The file content type is " + fileContentType, HttpStatus.BAD_REQUEST);
```

Comment:
Is it necessary to show the content type? I understand this is good for debugging but do we need to show it to the users

Reply:
Updated